src/renderer/components/RecentConnections.tsx

```diff
@@ -1,19 +1,19 @@
 import { Link } from 'react-router-dom';
 import { Button, Table } from 'react-bootstrap';
 import { useState, useEffect } from 'react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import { Trash } from 'react-bootstrap-icons';
 import { ConnectionModel } from '../../db/Models';
+import EditForm from './EditForm';
 
 import '../scss/RecentConnections.scss';
 
 const RecentConnections = () => {
   const [connect, setConnect] = useState<ConnectionModel[]>([]);
+  const getConnections = async () => {
+    const connections = await window.connections.ipcRenderer.fetch();
+    setConnect(connections);
+  };
   useEffect(() => {
-    const getConnections = async () => {
```

Comment:
this change is unnecessary - getConnections only needs to be in scope for the useEffect

Reply:
never mind, I missed that it was used below

Reply:
We can remove the declaration that's in scope though I thought I had done that in my PR but guess not

Reply:
Oh wait im trolling lol

Reply:
right, is this useEffect even needed?

Reply:
Yeah the call to getConnections() needs to be inside a useEffect()